gst/nnstreamer/tensor_merge/gsttensormerge.c

```diff
@@ -373,50 +373,51 @@ gst_tensor_merge_sink_event (GstCollectPads * pads, GstCollectData * data,
 }
 
 /**
- * @brief Generate TensorConfig with TensorsConfig
+ * @brief Generate out TensorsConfig with in TensorsConfig
  * @param tensor_merge tensor merger
  * @param configs Tensors Config Data
```

Comment:
378~379 plz update description for  in_config / out_config

Reply:
Updated!! Thank you